install_files/ansible-base/roles/restrict-direct-access/templates/sshd_config

```diff
@@ -22,20 +34,33 @@ ChallengeResponseAuthentication no
 KerberosAuthentication no
 KerberosGetAFSToken no
 GSSAPIAuthentication no
-X11Forwarding no
-X11DisplayOffset 10
-PrintMotd no
-PrintLastLog yes
-TCPKeepAlive yes
-AcceptEnv LANG LC_*
 UsePAM no
 UseDNS no
+
+# Cipher selection
+
+Ciphers chacha20-poly1305@openssh.com,aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes128-ctr
```

Comment:
Do we really need `chacha20-poly1305@openssh.com`? As that is specially being used from the cheap phones which does not have good hardware, and our users will use Tails on proper hardware to ssh.